pkg/storaged/stratis-details.jsx

```diff
@@ -398,6 +547,56 @@ export const StratisPoolDetails = ({ client, pool }) => {
                         </DescriptionListDescription>
                     </DescriptionListGroup>
                     }
+                    { pool.Encrypted && client.features.stratis_crypto_binding &&
+                    <DescriptionListGroup>
+                        <DescriptionListTerm className="control-DescriptionListTerm">
+                            {_("storage", "Passphrase")}
+                        </DescriptionListTerm>
+                        <DescriptionListDescription>
+                            { key_desc ? cockpit.format(_("using key description \"$0\""), key_desc) : <i>{_("not configured")}</i> }
```

Comment:
We don't use `<i>` anywhere in cockpit. This should perhaps be a [badge](https://pf5.patternfly.org/components/badge) or a [label](https://pf5.patternfly.org/components/label)? But that's more of a @garrett question.

There's more occurrences of `<i>` in this PR, but let's first discuss it here at one place.

Reply:
We shouldn't use `<i>` anywhere. It didn't mean anything for years, but now just means "it's different". Browsers usually italicize the text, but not necessarily. Screen readers are a toss-up. You should either use `<em>` or not...

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i

And here? We shouldn't use either `<i>` or `<em>`.

(I was going to mention it in the video-based review, where I extracted a bunch of screenshots, and haven't posted yet.)